docs/source/dynamic-outputs.rst

```diff
@@ -0,0 +1,129 @@
+Dynamic Outputs
```

Comment:
this format is now outdated and should be updated for consistency (either here or in another PR)

Reply:
Will update and push this (I'll squash the commit into the one that introduced this)